website/siteConfig.js

```diff
@@ -16,7 +16,7 @@ const siteConfig = {
   projectName: "radicle-drips-docs",
   organizationName: "radicle-dev",
   headerLinks: [],
-  headerIcon: "img/drips-logo2.png",
+  headerIcon: "img/drips-logo3.png",
```

Comment:
Don't know if my eyes play tricks on me, but I think the logo appears a bit wide?

![image](https://user-images.githubusercontent.com/6360154/186132331-e1d40a39-ae93-49a8-8081-b87b107ac320.png)